app/helpers/application_helper/toolbar/middleware_server_center.rb

```diff
@@ -37,4 +38,27 @@ class ApplicationHelper::Toolbar::MiddlewareServerCenter < ApplicationHelper::To
       ]
     ),
   ])
+  button_group('middleware_server_operations', [
+    select(
+      :middleware_server_power_choice,
+      'fa fa-power-off fa-lg',
```

Comment:
I'm less familiar with toolbars design, is it ok to pass here hardcoded in server code the icons?